docker-compose.test.yml

```diff
@@ -0,0 +1,72 @@
+version: "3.7"
+
+x-common-variables: &test-variables
+  HOST: http://testapp
+  COOKIE_NAME: _proxy
+  TOKEN_SECRET: Rm9yIEdvZCBzbyBsb3ZlZCB0aGUgd29ybGQgdGhhdCBoZSBnYXZlIGhpcyBvbmUgYW5kIG9ubHkgU29uLCB0aGF0IHdob2V2ZXIgYmVsaWV2ZXMgaW4gaGltIHNoYWxsIG5vdCBwZXJpc2ggYnV0IGhhdmUgZXRlcm5hbCBsaWZlLiAtIEpvaG4gMzoxNg==
```

Comment:
I'm surprised this didn't trigger a warning that a secret was committed. I wonder if we can use something simpler here.